tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -64,7 +64,9 @@ def generate_random_inputs(
     return input_values
 
 
-def check_correctness(model: ModelProto, inputs: Optional[Dict[str, np.array]] = None, opset: int = None) -> None:
+def check_correctness(
+    model: ModelProto, inputs: Optional[Dict[str, np.array]] = None, opset: int = None
+) -> None:
```

Comment:
I assume stuff like this is linting fixes? It's best to keep these in a separate PR if they're linting unrelated to the change, otherwise the diff becomes a bit difficult to follow.

Reply:
I removed the lint fixes.